Makefile

```diff
@@ -111,9 +111,7 @@ clean:
 		test/copyimg/copyimg \
 		test/testdata/redis-image \
 		$(MANPAGES)
-ifdef HAS_PYTHON3
```

Comment:
PYTHON will == /usr/bin/python3 ... what happens if that is not present like on CentOS?

Reply:
the problem is that varlink python requires python3 ... so in this case, it is important to be python3.  I'm not trying to be a PITA, but this is why i did what I did.  There doesnt seem to be an ideal approach until this code it punted to its own repo.

one thing we did in the spec files was to focus on varlink instead of python3.  so we had a bool for with_varlink and that include f28+ ... maybe a similar concept could be applied here?

Reply:
> the problem is that varlink python requires python3 ... so in this case, it is important to be python3.

But *this* case is `clean`, which does not require Python 3 (as the invocation in my [initial comment here][1] shows).  This PR does not touch [the `HAS_PYTHON3` check for the `python-podman` target][2], because there we do actually need Python 3 for a successful call.

[1]: https://github.com/projectatomic/libpod/pull/818#issue-189446284
[2]: https://github.com/projectatomic/libpod/blob/82feafecdda8040432c008d9b79e4f973009adfc/Makefile#L98-L101